ui/pages/admin/users.jsx

```diff
@@ -1,35 +1,41 @@
 // TODO [tech]
 import React, { useEffect, useState } from "react";
-import { _delete, _get, _post, _put } from "../../common/httpClient";
 import { useFormik } from "formik";
+import { useQuery } from "@tanstack/react-query";
+import Head from "next/head";
+import generator from "generate-password-browser";
 import {
   Accordion,
   AccordionButton,
   AccordionIcon,
   AccordionItem,
   AccordionPanel,
+  Badge,
   Box,
   Button,
   Checkbox,
+  Flex,
   FormControl,
   FormLabel,
   Heading,
   HStack,
   Input,
   Stack,
-  useToast,
+  Table,
+  Td,
   Text,
+  Tr,
+  VStack,
 } from "@chakra-ui/react";
-import { Breadcrumb } from "../../components/Breadcrumb/Breadcrumb";
-import generator from "generate-password-browser";
-import { useQuery } from "@tanstack/react-query";
-import { Page } from "../../components/Page/Page";
-import Head from "next/head";
-import withAuth from "../../components/withAuth";
 
-import Acl from "../../components/Acl";
-import { getAuthServerSideProps } from "../../common/SSR/getAuthServerSideProps";
-import { Check } from "../../theme/components/icons";
+import { _delete, _get, _post, _put } from "@/common/httpClient";
+import { Breadcrumb } from "@/components/Breadcrumb/Breadcrumb";
+import { Page } from "@/components/Page/Page";
+import withAuth from "@/components/withAuth";
+import Acl from "@/components/Acl";
```

Comment:
Si vous voulez partir sur des alias ok. 

perso je suis pas un grand fan pour des questions de portabilité entre projets car ça force à avoir de la conf plus haut niveau. Après dans la pratique c'est peut arrivé qu'il n'y ait pas de jsconf au root dir. why not

Reply:
c surtout pour éviter les chemins relatifs à rallonge.